src/components/views/elements/AccessibleButton.js

```diff
@@ -27,8 +27,12 @@ import React from 'react';
 export default function AccessibleButton(props) {
     const {element, onClick, children, ...restProps} = props;
     restProps.onClick = onClick;
-    restProps.onKeyUp = function(e) {
-        if (e.keyCode == 13 || e.keyCode == 32) return onClick(e);
+    restProps.onKeyDown = function(e) {
```

Comment:
Perhaps document here why it needs to be onKeyDown, and what is special about keyCode 13 and 32

Reply:
Found a related issue so done along with its fix in 839f938